shared/instance.go

```diff
@@ -1206,6 +1206,15 @@ func ConfigKeyChecker(key string, instanceType instancetype.Type) (func(value st
 			return validate.IsAny, nil
 		}
 
+		// lxdmeta:generate(entity=instance, group=volatile, key=volatile.<name>.<target_path>.dir_created_by_lxd)
+		//
+		// ---
+		//  type: bool
+		//  shortdesc: if the disk device mounted from a host directory, created a new directory within the instance
+		if strings.HasSuffix(key, ".dir_created_by_lxd") {
```

Comment:
I feel `dir_created_by_lxd` looks to be quite long compared to the other volatile keys. Maybe we could just do `dir_created`?